### PR TITLE
Add color argument to makeSceneImage() function

### PR DESCRIPTION
**Pull request by vishesh**

### CODE REVIEW

**Comment by soegaard** [comment]
Thanks.

File: whalesong/image/private/js-impl.js [hunk 1]
```diff
@@ -921,10 +924,11 @@ EXPORTS['scene+line'] =
 	    var y2 = checkReal(MACHINE,	"scene+line", 4);
 	    var c = checkColor(MACHINE, "scene+line", 5);
 	    // make a scene containing the image
-	    var newScene = makeSceneImage(jsnums.toFixnum(img.getWidth()), 
-                                    jsnums.toFixnum(img.getHeight()),
-                                    [],
-                                    false);
+            var newScene = makeSceneImage(jsnums.toFixnum(img.getWidth()), 
+                                          jsnums.toFixnum(img.getHeight()),
+                                          null,
```
Comment (stchang) [inline whalesong/image/private/js-impl.js]: Should this call pass in the `c` color instead of `null`?

Comment (vishesh) [inline whalesong/image/private/js-impl.js]: The color parameter is for the line. This scene acts as container, over which we will render line.

Comment (stchang) [inline whalesong/image/private/js-impl.js]: Ah ok, I got it now. Thanks.

**Pull request merged**